src/coreclr/scripts/superpmi.py

```diff
@@ -2981,6 +2992,7 @@ def replay_with_throughput_diff(self):
                 ]
                 flags = [
                     "-applyDiff",
+                    "-v", "ewi",
```

Comment:
what additional information do we get with this that would help, which is not present easily today?

Reply:
I'm not sure, but `asmdiffs` and `replay` both specify these as well, so I figured `tpdiff` also should.